### PR TITLE
add footer credit and tagline and rework intro

### PR DESCRIPTION
**Pull request by ryanatkn**
- adds a footer section crediting Felt Co-op as the operator (I'm not sure about that word (maybe "stewarded"?), and I'm wondering if a link to the governance doc belongs here)
- reworks the intro trying to be shorter and clearer (it looks redundant now, but I think we need a tagline in the subheader, not the current text, with that change it won't be redundant)
- replace the tagline ("web tools that feel good")

### CODE REVIEW

File: src/routes/Footer.svelte [hunk 1]
```diff
@@ -3,6 +3,13 @@
 </script>
 
 <footer>
+	<div class="panel markup">
+		<div>operated by Felt Co-op</div>
```
Comment (ryanatkn) [inline src/routes/Footer.svelte:7]: "stewarded"? something else?

Comment (greatbacon) [reply src/routes/Footer.svelte:7]: I like stewarded, and I don't think a link to the gov'n doc would hurt.

**Review by greatbacon — COMMENTED** [review]
Looks good so far! I'll hold off on approving til' we talk about the tagline

**Review by greatbacon — APPROVED** [review]
💚

**Pull request merged**